src/TokenizedStrategy.sol

```diff
@@ -498,15 +502,16 @@ contract TokenizedStrategy {
         uint256 assets,
         address receiver
     ) external nonReentrant returns (uint256 shares) {
+        StrategyData storage S = _strategyStorage();
```

Comment:
So TLDR, external/public functions load the point, internal/private functions receive the pointer?

Reply:
yup